changelogs/current.yaml

```diff
@@ -32,6 +32,9 @@ minor_behavior_changes:
 - area: rate_limit
   change: |
     add ``MONTH`` and ``YEAR`` to the unit of time for rate limit.
+- area: stats
```

Comment:
Please move to `behavior_changes` section, and document the old and new names, similar to https://github.com/envoyproxy/envoy/blob/3752119b94ae4a5ff337364f585f54305406dc7f/changelogs/1.24.0.yaml#L8